libavformat/aviobuf.c

```diff
@@ -1023,7 +1023,24 @@ URLContext* ffio_geturlcontext(AVIOContext *s)
 int ffio_copy_url_options(AVIOContext* pb, AVDictionary** avio_opts)
 {
     const char *opts[] = {
```

Comment:
Do we want _all_ the `http.c` options?

Reply:
Right now this code is only used  in decode/read, I think I got all the "read" related HTTP options